src/ipsis/course-merge/CourseMergeLogDetailCollectionEntity.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * CourseMergeLogDetailCollectionEntity class representation of course merge log collection as defined in the LMS
+ * See: ISirenCourseMergeSerializer.SerializeCourseMergeLogDetailsListResult
+ */
+import { Entity } from '../../es6/Entity.js';
+
+export class CourseMergeLogDetailCollectionEntity extends Entity {
```

Comment:
Might be useful to extend from `BaseCollectionEntity` in the future, but for now while we don't have load more, I think this is fine